priv/repo/seeds.exs

```diff
@@ -18,6 +18,7 @@ Repo.transaction(fn ->
     "external_id" => tenant_name,
     "jwt_secret" =>
       System.get_env("API_JWT_SECRET", "super-secret-jwt-token-with-at-least-32-characters-long"),
+    "max_channels_per_client" => Application.get_env(:realtime, :max_channels_per_client, 100),
```

Comment:
```suggestion
```

with the change suggested of settings it as a default value of the tenant we can remove it from here as seed will be able to use said value